### DISCUSSION TITLE
Does the IndexOptimizer optimize all databases, or just master?

### DISCUSSION THREAD

**Discussion by padrickr** [olahallengren/sql-server-maintenance-solution#754] [General]
I want to run this against my MECM (SCCM) database, so would I need to specify that somewhere?

**Comment by SysDaemon** [comment]
The script optimize all databases, or just a database you want to.

See in [SQL Server Index and Statistics Maintenance](https://ola.hallengren.com/sql-server-index-and-statistics-maintenance.html),
on parameter "**Databases**" to see it in more details.

**Reply by padrickr** [reply]
Great, thank you.
